Lib/test/test_mimetypes.py

```diff
@@ -305,6 +306,24 @@ def test_keywords_args_api(self):
         self.assertEqual(self.db.guess_extension(
             type='image/jpg', strict=False), '.jpg')
 
+    def test_added_types_are_used(self):
+        mime_type, _ = mimetypes.guess_type('test.myext')
+        self.assertEqual(None, mime_type)
+        mimetypes.add_type('testing/type', '.myext')
+        mime_type, _ = mimetypes.guess_type('test.myext')
+        self.assertEqual('testing/type', mime_type)
```

Comment:
Nit: asserts are usually written with test output first, expected result second:

```suggestion
        self.assertEqual(mime_type, None)
        mimetypes.add_type('testing/type', '.myext')
        mime_type, _ = mimetypes.guess_type('test.myext')
        self.assertEqual(mime_type, 'testing/type')
```